app/assets/stylesheets/components/_govspeak-editor.scss

```diff
@@ -4,26 +4,32 @@
 
 .app-c-govspeak-editor__preview-button-wrapper {
   text-align: right;
+  display: flex;
+  justify-content: end;
+  gap: govuk-spacing(2);
```

Comment:
Lovely to see the `gap` attribute in the wild, was waiting in anticipation for that last time I did any serious frontend work